### ISSUE TITLE
Allow adding new work items to an existing session

### ISSUE THREAD

**Issue by cschleiden** [cschleiden/vsts-estimate#1]

**Comment by XristophD** [comment]
Hey, this is the main reason we are currently looking for an alternate tool for estimations. 
Resending the link because some items changed in the backlog is a bit cumbersome for bigger teams.

**Comment by cschleiden** [comment]
This is still planned, however, the problem is not adding new work items to a session, that's fairly easy to build. The difficulty is somehow selecting the work items. The product doesn't expose an easily resuable "WorkItemPicker", for example, so that's something that would have to be built. 

What do you think would be a good experience for selecting the additional work items? 

For this:

> Resending the link because some items changed in the backlog is a bit cumbersome for bigger teams.

In version 2, if you start an estimation based on a team's iteration or on a query, the work items are loaded fresh each time you enter the session. Is that how you typically start your sessions or do you explicitly select work items and then use the context menu to start?

**Comment by XristophD** [comment]
Hey. thank you a lot for the fast response and the hint to the new functionality for the 2.0 Preview version about working with queries and refreshing the session with the same session link. This already solves out problem and we can now work exactly as we wanted with the tool. 

About adding new backlog items during the estimation: we do this now on the backlog board, move it to the column our query is set to (approved) and ask the team to refresh their browser. so, problem solved at the moment. :-) 

Thanks for the great work here!

**Comment by cschleiden** [comment]
I guess adding a refresh button in the overflow to re-run the query/fetch the iteration for all users would be a nice addition, too. Filed #32

**Comment by DrewRhoades** [comment]
I just want to right-click on a work item in  the backlog and pick something like "Add to Existing Estimate Session".

**Comment by Babiarzmt** [comment]
I agree with @DrewRhoades the right click addition would be an ideal solution.
